rtb/verbali/esterni/VE_2024-11-27_ver1.0.0.typ

```diff
@@ -0,0 +1,77 @@
+#import "/templates/template.typ": *
+
+#show: content=>verbale(
+  data: "27 novembre 2024", 
+  responsabile: "Davide Picello", 
+  destinatari: (
+    "Prof. Tullio Vardanega",
+    "Prof. Riccardo Cardin",
+    "Sync Lab S.r.l."
+  ),
+  uso: "Esterno",
+  redattori: ("Davide Picello",), 
+  titolo: "SAL fine primo sprint", 
+  verificatori: (
+    "Riccardo Milan",
+  ),
+  riassunto: "Incontro di fine sprint dove sono stati chiariti molti dubbi, e fissati alcuni obiettivi per lo sprint a venire",
+  versioni: (
+    "1.0.0", 
+    [28/11/2024], 
+    "Davide Picello", 
+    "Riccardo Milan",
+    [- Stesura del verbale],
+  ),
+  content: content,
+  firmaRichiesta: true,
+)
+
+
+#infoRiunione(luogo: "Google Meet", data: "27/11/2024", ora: "16:10", durata: "40 minuti", partecipantiEsterni: ("Fabio Pallaro", "Andrea Dorigo"))
```

Comment:
Google Meet in corsivo

Reply:
le NdP dicono:

### Corsivo col comando \_termine\_:
- Termini in lingua inglese (esclusi se si tratta di nomi di prodotti software o simili)

Quindi penso qui non serva

Reply:
ok